scrapers/NaughtyAmerica.yml

```diff
@@ -27,9 +27,9 @@ xPathScrapers:
       Studio:
         Name: $sceneinfo/a[@class="site-title grey-text link"]
       Image:
-        selector: //img[@class="start-card lazyload"]/@data-src|//img[@class="start-card"]/@src|//dl8-video[@id="vr_player"]/@poster|(//div[@class="overlays"]/a/picture)[1]/source[@type="image/jpeg"]/@data-srcset
+        selector: //img[@class="start-card lazyload"]/@data-src|//img[@class="start-card"]/@src|//dl8-video[@id="vr_player"]/@poster|(//div[@class="overlays"]/a/picture)[1]/source[@type="image/jpeg"]/@data-srcset|//div[@class="contain-scene-images desktop-only"]/a[2]/@href
```

Comment:
The added selector `//div[@class="contain-scene-images desktop-only"]/a[2]/@href` for the VR scenes  gets the 2nd image from the gallery part
That is not the same with the poster(main) image
The below selector seems more accurate to me
`//img[contains(@class,"start-card desktop-only")]/@data-srcset`
as ti gets the proper image

Reply:
Perfectly acceptable to me. I was shooting for the second as it didn't have all the extra banner garbage but I can see how that might be preferred.  I'll go ahead and update.

Reply:
If that is the case we can add the selector commented out so that users can choose what they need.
In that case though would you want the second image for all the scenes or just the VR ones?